Orisabiyi/luna
# Luna Inovice Creation

# Pull Request: Implement Invoice Creation Feature

## Description
This pull request implements invoice creation feature for the luna platform.
The following functionality are available: allowing users to connect with the Invoicing Smart Contract, create invoices, and receive real-time feedback on transaction status.

## Current Feature
- Implement component for creating new invoices
- Added loader component to display during blockchain transactions
- Integrated component for success and error messages
- Connected frontend with Invoicing Smart Contract using Stacks.js
- Implemented real-time calculation of invoice totals
- Added ability to dynamically add, edit, and delete line items in invoices

## How to Test
1. Clone this branch
2. Install dependencies: `pnpm install`
3. Start the development server: `pnpm dev` or `pnpm run dev`
4. Navigate to the invoice creation page
5. Try creating an invoice with multiple line items
6. Verify that the loader appears during transaction processing
7. Confirm that a success notification appears after successful invoice creation

## Checklist
- [x] Authentication of users using StacksJS
- [x] Edit and Updating of available invoice template
- [x] RealTime feedback of transaction
- [x] README.md Documentation for the current version of the platform
